main.js

```diff
@@ -0,0 +1,15 @@
+
+let displayScreen = document.querySelector('#display')
+console.log(displayScreen)
+
+let buttonClear = document.querySelector('#clear')
+console.log(buttonClear)
+
+let buttonNumbers = document.querySelectorAll(".buttonNum")
+console.log(buttonNumbers)
+
```

Comment:
This is the right approach for selecting multiple items with the same class. Note that you will want to loop through all of these to add an event listener to each one. Check out the DEMO from April 20 for an example. https://momentum-team-18.github.io/course/